DuoUniversal/Labels.cs

```diff
@@ -24,6 +24,7 @@ internal class Labels
         public const string RESPONSE_TYPE = "response_type";
         public const string SCOPE = "scope";
         public const string STATE = "state";
+        public const string ISSUER = "issuer";
```

Comment:
OIDC already has a concept of an "issuer" but it only applies to the responses coming from the server - nobody is expecting the client to send it.  Is this the same concept?  if so, we should call it by the same name "iss".  If it's something else, maybe we can try a different name so it doesn't get confused?

Reply:
This issuer is actually a SAML Audience. It sounds weird, but this issuer parameter's value gets assigned to the audience in Trusted Path with the current Epic authentication flow. So we shouldn't treat it as an OIDC issuer.  Another name? audienceIssuer? I guess we should not use the customer name in the parameters' names :)